doc-content/enterprise-only/openshift/offline-repo-proc.adoc

```diff
@@ -51,7 +65,13 @@ endif::DM[]
 place your services in this repository as well. You must configure this repository as the external Maven repository. You cannot configure a separate mirror repository in an immutable environment.
 endif::[]
 ifeval::["{context}"!="openshift-immutable"]
-. Prepare a Maven release repository to which you can write. The repository must allow read access without authentication. Your OpenShift environment must have access to this repository. You can deploy a Nexus repository manager in the OpenShift environment. For instructions about setting up Nexus on OpenShift, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/developer_guide/tutorials#nexus-setting-up-nexus[Setting up Nexus]. Use this repository as a separate mirror repository.
+. Prepare a Maven release repository to which you can write. The repository must allow read access without authentication. Your OpenShift environment must have access to this repository. 
```

Comment:
Rewrite suggestion:
Prepare a Maven release repository with write access. The repository must allow read access without authentication. and your OpenShift environment must have access to this repository. 

Also, are you really preparing the repos or configuring it?